launch/app/app.bndrun

```diff
@@ -223,9 +223,11 @@ feature.openhab-model-runtime-all: \
 	org.apache.servicemix.specs.annotation-api-1.3;version='[1.3.0,1.3.1)',\
 	org.apache.servicemix.specs.jaxb-api-2.2;version='[2.9.0,2.9.1)',\
 	org.apache.servicemix.specs.stax-api-1.2;version='[2.9.0,2.9.1)',\
-	org.openhab.transform.map;version='[2.5.5,2.5.6)',\
-	org.openhab.ui.basic;version='[2.5.5,2.5.6)',\
-	org.openhab.ui.dashboard;version='[2.5.5,2.5.6)',\
-	org.openhab.ui.iconset.classic;version='[2.5.5,2.5.6)',\
-	org.openhab.ui.paper;version='[2.5.5,2.5.6)',\
-	org.openhab.ui.restdocs;version='[2.5.5,2.5.6)'
+	org.glassfish.hk2.external.javax.inject;version='[2.4.0,2.4.1)',\
+	org.glassfish.hk2.utils;version='[2.4.0,2.4.1)',\
```

Comment:
They don't seems to be necessary, so if you add them to the `-runblacklist` they will never be added again by the resolver.